### PR TITLE
#907 New Scripts - UTILITIES - Elig Results to Word

### PR DESCRIPTION
**Pull request by C-Love**
New script to create a word document of a single Eligibility Result for any program other than HC. This is supportive of processes such as appeals, overpayments or other claims activities.

I will update CLoS in another pull since this branch is quite old.

### CODE REVIEW

**Review by courtrightd — COMMENTED** [review]
Pushed a commit that fixes an error on GA results, tested good otherwise. 
Might want to adjust the date on changelog initial version.

**Comment by MeganGeissler** [comment]
Ran for ELIG/EMER and it kept rewriting EMSM results. I think you need one more handling for worker_message: that ends with " **"
If worker_message = "** PLEASE PROVIDE A COMMAND OR PF-KEY TO CONTINUE **" Then last_elig_screen = True

ELIG word doc

File: utilities/elig-results-to-word.vbs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+		If InStr(MX_line_3, "DWPR") Then elig_results_program_found = "DWP"
+		If InStr(MX_line_3, "MFPR") Then elig_results_program_found = "MFIP"
+		If InStr(MX_line_3, "MSPR") Then elig_results_program_found = "MSA"
+		If InStr(MX_line_3, "GAPR") Then elig_results_program_found = "GA"
+		If InStr(MX_line_3, "CAPR") Then elig_results_program_found = "Cash Denial"
+		If InStr(MX_line_3, "GRPR") Then elig_results_program_found = "GRH"
+		If InStr(MX_line_3, "FCSM") Then elig_results_program_found = "IV-E"
+		If InStr(MX_line_3, "EMPR") Then elig_results_program_found = "EMER"
+		If InStr(MX_line_3, "FSPR") Then elig_results_program_found = "SNAP"
+		If elig_results_program_found = "" Then MsgBox "MAXIS must be at the first page of Eligibility Results for this script to run." & vbCr & vbCr & "The dialog will return." & vbCr & vbCr & "NAVIGATE TO ELIG RESULTS WHILE THE DIALOG IS UP."
```
Comment (MDR22) [inline utilities/elig-results-to-word.vbs:118]: Not sure if it's necessary but wondered about handling to ensure that the script is pulling results for the case number initially entered. I put in a case number, received the notice to move to elig results, but then navigated to elig results for a different case number. It printed the results for the new case number as expected, but thought about whether an error message of some sort might be useful if the elig results are for a different case number.

**Review by MDR22 — APPROVED** [review]
Looks good!

**Review by IlseFerris — CHANGES_REQUESTED** [review]
Good feedback by @MeganGeissler and @MDR22. I don't have any additional comments besides theirs.

**Comment by C-Love** [comment]
I think with all the variations in the last page handling, we needed something more dynamic - so I updated this to using an InStr.

File: utilities/elig-results-to-word.vbs (hunk 1 above)
Comment (C-Love) [reply utilities/elig-results-to-word.vbs:111]: Good thought - I think with this script we don't really care what the case number is. Removing it entirely from the dialog.

**Review by IlseFerris — APPROVED** [review]
Tested on FS and MFIP results and working as intended. I approve thee!

**Pull request merged**